tests/pytests/test_json.py

```diff
@@ -581,7 +581,8 @@ def testDifferentType(env):
     env.expect('FT.SEARCH', 'hidx', '*', 'NOCONTENT').equal([1L, 'doc:1'])
     env.expect('FT.SEARCH', 'jidx', '*', 'NOCONTENT').equal([1L, 'doc:2'])
 
-def test_NoSortableArray(env):
+def test_NoSortableJsonTag(env):
+    env.skipOnCluster()
     conn = getConnectionByEnv(env)
     env.execute_command('FT.CREATE', 'idx', 'ON', 'JSON', 'SCHEMA', '$.tag[*]', 'TAG', 'SORTABLE')
```

Comment:
So why not raise an error if you combine sortable with tag field on json?

Reply:
We didn't raise this error for GEO before. Just ignored it.

Reply:
Fixed